app-frontend/src/app/components/exports/exportDownloadModal/exportDownloadModal.component.js

```diff
@@ -0,0 +1,15 @@
+// Component code
+import exportDownloadModalTpl from './exportDownloadModal.html';
+
+const ExportDownloadModalComponent = {
+    templateUrl: exportDownloadModalTpl,
+    bindings: {
+        close: '&',
```

Comment:
We have this on all of our other modals, a bit leery of removing it. Will probably just keep it.